scripts/travis/deploy_packages.sh

```diff
@@ -12,10 +12,6 @@ set -e
 
 SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
 
-if ! "${SCRIPTPATH}/../check_golang_version.sh"
```

Comment:
Instead of deleting this, should we move it after the gimme, to verify that it installed successfully?

Reply:
I'm not sure the script even needs to do the check.

Reply:
Re-added it after the check.